admin/modules/user/users.php

```diff
@@ -1011,9 +1011,10 @@
 		}
 	}
 	$avatar_top = ceil((126-$scaled_dimensions['height'])/2);
-	if($user['lastactive'])
+	$last_seen = max(array($user['lastactive'], $user['lastvisit']));
```

Comment:
Super minor, but this can be written as `max($user['lastactive'], $user['lastvisit'])` - I'm not going to block the PR for this though, it's justa  note 😄